src/types/users.ts

```diff
@@ -23,7 +23,7 @@ export type UserNotifyProps = {
 };
 
 export type UserProfile = {
-    id: string;
+    id: 'me' | string;
```

Comment:
Is there any reason to have this distinction here?

Reply:
nope. will remove